ocaml/libs/tracing/dune

```diff
@@ -31,6 +31,7 @@
 (library
  (name tracing_propagator)
  (modules propagator)
+ (wrapped false)
```

Comment:
Please avoid using unwrapped libraries, it's an annoying source of headaches because it makes it difficult to understand whhich library do modules come from.